### DISCUSSION TITLE
Chromebook Edition projects

### DISCUSSION THREAD

**Discussion by DennisLfromGA** [Morpheus636/awesome-framework#20] [RFC]
I am soon to get (Batch 1) the Framework Laptop Chromebook Edition and plan to put it in Developer mode and try some stuff. ChromeOS is still pretty locked down so I don't know how much I'll be able to do.

I look forward to seeing some projects, tools, documentation and resources related to the Chromebook Editon too.

~DennyL

**Comment by Morpheus636** [comment]
Awesome! I haven't added anything Chromebook Edition specific yet as nobody has one in their hands yet, but once there are some projects out there I would be more than happy to add them. Keep us updated on what you figure out.
